docs/framework/whats-new/whats-new-in-accessibility.md

```diff
@@ -0,0 +1,255 @@
+    ![Toolbar.ComboBoxStyleKey before accessibility improvements](media/comboboxstylekey-before.png) 
+    
+    After: 
+
+    ![Toolbar.ComboBoxStyleKey after accessibility improvements](media/comboboxstylekey-after.png) 
+
+- <xref:System.Windows.Controls.DataGrid> control
+
+    Starting with the .NET Framework 4.7.1, the sort indicator arrow in <xref:System.Windows.Controls.DataGrid> controls now uses correct theme colors. For example:
+
+    Before: 
+
+    ![Sort indicator arrow before accessibility improvements](media/sort-indicator-before.png) 
+    
+    After:   
+ 
+    ![Sort indicator arrow after accessibility improvements](media/sort-indicator-after.png) 
+    
+    In addition, in the .NET Framework 4.7 and earlier versions, the default link style changed to an incorrect color on mouse over in high contrast modes. This is resolved starting with the .NET Framework 4.7.1. Similarly, <xref:System.Windows.Controls.DataGrid> checkbox columns uses the expected colors for keyboard focus feedback starting with the .NET Framework 4.7.1.
+
+    Before: 
+
+    ![DataGrid default link style before accessibility improvements](media/default-link-style-before.png) 
+ 
+    After:    
+  
+    ![DataGrid default link style after accessibility improvements](media/default-link-style-after.png)  
+
+For more information on WPF accessibility improvements in the .NET Framework 4.7.1, see [Accessibility improvements in WPF](../framework/migration-guide/retargeting/4.7-4.7.1.md#accessibility-improvements-in-wpf).
+
+## Windows Forms accessibility improvements
+
+In the .NET Framework 4.7.1, Windows Forms (WinForms) includes accessibility changes in the following areas.
+
+**Improved display in High Contrast mode**
+
+Starting with the .NET Framework 4.7.1, various WinForms controls offer improved rendering in the HighContrast modes available in the operating system. Windows 10 has changed the values for some high contrast system colors, and Windows Forms is based on the Windows 10 Win32 framework. For the best experience, run on the latest version of Windows and opt in to the latest OS changes by adding an app.manifest file in a test application and un-comment the Windows 10 supported OS  line so that it looks the following:
+
+```xml
+<!– Windows 10 –>
+<supportedOS Id=”{8e0f7a12-bfb3-4fe8-b9a5-48fd50a15a9a}” />
+```
+Some examples of high contrast changes include:
+
+- Checkmarks in <xref:System.Windows.Forms.MenuStrip> items are easier to view.
+
+- When selected, disabled <xref:System.Windows.Forms.MenuStrip> items are easier to view.
+
+- Text in a selected <xref:System.Windows.Forms.Button> control contrasts with the selection color.
+
+- Disabled text is easier to read. For example:
+
+    Before:
+
+    ![Disabled text before accessibility improvements](media/wf-disabled-before.png) 
+
+    After:
+
+    ![Disabled text after accessibility improvements](media/wf-disabled-after.png) 
+- High contrast improvements in the Thread Exception Dialog.
```

Comment:
Would a before/after image be helpful here?

Reply:
There is a before/after image here. Sorry, @merriemcgaw  -- I should have included a link to the staged published topic. It is [What's new in accessibility in the .NET Framework](https://review.docs.microsoft.com/en-us/dotnet/framework/whats-new/whats-new-in-accessibility?branch=pr-en-us-3522).